src/components/grid/GridComponent.vue

```diff
@@ -58,7 +58,7 @@
               </span>
             </th>
             <td>
-              <button class="ll-facet-option btn btn-sm selectRow gridItem btn-outline-secondary"
```

Comment:
`select-row` looks better than `selectRow` but `selectAll, selectItem, selectCol` should than probably have the same treatment?